matRad_generateStf.m

```diff
@@ -322,8 +332,16 @@
         stf(i).numOfRays = size(stf(i).ray,2);
         for j = 1:stf(i).numOfRays
             stf(i).numOfBixelsPerRay(j) = numel(stf(i).ray(j).energy);
+            % get for each spot the focus index
             for k = 1:stf(i).numOfBixelsPerRay(j)
-                stf(i).ray(j).focusIx(k) = 
+                 energyIx = find(round2(stf(i).ray(j).energy(k),4) == round2([machine.data.energy],4));
```

Comment:
Why do we need a rounding operation here? Isnt the stf.ray.energy directly constructed out of machine.data.energy? There should be a perfect match?!